CoreScriptsRoot/Modules/VR/NotificationHub.lua

```diff
@@ -0,0 +1,723 @@
+local CoreGui = game:GetService("CoreGui")
+local RobloxGui = CoreGui:WaitForChild("RobloxGui")
+local Util = require(RobloxGui.Modules.Settings.Utility)
+local Panel3D = require(RobloxGui.Modules.VR.Panel3D)
+local VRHub = require(RobloxGui.Modules.VR.VRHub)
+
+local thisModuleName = "Notifications"
+
+local NO_TRANSITION_ANIMATIONS = false
+local ANIMATE_OUT_DISTANCE = -50
+local PIXELS_PER_STUD = 150
+local WINDOW_TITLEBAR_HEIGHT = 72
+local BLURRED_TITLEBAR_COLOR = Color3.new(78 / 255, 84 / 255, 96 / 255)
+local FOCUSED_TITLEBAR_COLOR = Color3.new(82 / 255, 101 / 255, 141 / 255)
+local WINDOW_BG_COLOR = Color3.new(20/255, 20/255, 20/255)
+local WINDOW_BG_TRANSPARENCY = 0.5
+local POPOUT_DISTANCE = 0.25
+
+local NOTIFICATION_WIDTH_SCALE = 0.85
+local NOTIFICATION_HEIGHT_OFFSET = 80
+local NOTIFICATION_PADDING_Y = 20
+local NOTIFICATION_PADDING_X_SCALE = (1 - NOTIFICATION_WIDTH_SCALE) / 2
+local NOTIFICATION_DEPTH_OFFSET = 0.25
+local MAX_NOTIFICATIONS_SHOWN = 3
+local MAX_DETAILS_SHOWN = 2
+local DETAILS_PADDING = 20
+
+local BUTTON_NORMAL_IMG = "rbxasset://textures/ui/Settings/MenuBarAssets/MenuButton.png"
+local BUTTON_SELECTED_IMG = "rbxasset://textures/ui/Settings/MenuBarAssets/MenuButtonSelected.png"
+
+local emptySelectionImage = Util:Create "ImageLabel" {
+	BackgroundTransparency = 1,
+	Image = ""
+}
+
+
+local AVATAR_IMAGE_URL = 'http://www.roblox.com/thumbs/avatar.ashx?userId=%d&x=%d&y=%d'
+
+local TEXT_COLOR = Color3.new(1, 1, 1)
+
+local aspectRatio = 1.62666666
+local totalHeight = 3.5
+local totalWidth = totalHeight * aspectRatio
+local leftPanelWidth = totalWidth * 0.4
+local rightPanelWidth = totalWidth * 0.6
+
+local panelOffset = totalWidth / 2
+local leftOffset = panelOffset - (leftPanelWidth * 0.5)
+local rightOffset = leftOffset - (leftPanelWidth * 0.5) - (rightPanelWidth * 0.5)
+
+local NotificationHubModule = {}
+NotificationHubModule.ModuleName = "Notifications"
```

Comment:
Can we either assign the thisModuleName variable to NotificationHubModule.ModuleName, or use NotificationHubModule.ModuleName in places where thisModuleName is used. If we ever change the name we'll have to remember to change it both here and above on line 7